### PR TITLE
feat: add a editorconfig icon

### PR DESCRIPTION
**Pull request by r3nya**
Added EditorConfig icon.
@oivva please review it.

### CODE REVIEW

File: src/icons.json [hunk 1]
```diff
@@ -213,6 +213,19 @@
       }
     ]
   },
+  "file_type_editorconfig": {
```
Comment (oivva) [inline src/icons.json]: Before creating an alias search for syntax on Package Control. In this case syntax provided by https://github.com/sindresorhus/editorconfig-sublime/blob/master/EditorConfig.tmLanguage

```js
  "file_type_editorconfig": {
    "color": "yellow",
    "syntaxes": [
      {
        "name": "EditorConfig",
        "scope": "source.ini.editorconfig"
      }
    ]
  },
```

Comment (oivva) [inline src/icons.json]: We are always trying to minimize alias abuse.

Comment (r3nya) [inline src/icons.json]: Oh, sorry. It's my bad.

Comment (oivva) [inline src/icons.json]: In this case user have to install `EditorConfig` package to see its icon.

Comment (r3nya) [inline src/icons.json]: Done!

**Comment by oivva** [comment]
Nice work, thanks!

File: src/icons.json (hunk 1 above)
Comment (r3nya) [inline src/icons.json]: Btw, now I understood about syntax scopes and other things. Thanks. :)

Comment (oivva) [inline src/icons.json]: @r3nya Cool) Yes, the entire process is pretty clear after adding a few icons :wink: Also recommended to not provide `px` in svg width & height, I'll remove them, cause I've already merged.

Comment (r3nya) [inline src/icons.json]: @oivva deal! :)

**Pull request merged**